src/data/constants.js

```diff
@@ -0,0 +1,50 @@
+import { getConfig } from '@edx/frontend-platform';
+
+export const API_BASE_URL = getConfig().LMS_BASE_URL;
+
+export const LoadingStatus = {
+  LOADING: 'loading',
+  LOADED: 'loaded',
+  FAILED: 'failed',
+  DENIED: 'denied',
+};
+
+export const ThreadOrdering = {
+  BY_LAST_ACTIVITY: 'sort_by_last_activity',
+  BY_COMMENT_COUNT: 'sort_by_comment_count',
+  BY_VOTE_COUNT: 'sort_by_vote_count',
+};
+
+export const ThreadView = {
+  UNREAD: 'unread',
+  UNANSWERED: 'unanswered',
+};
+
+export const MyPostsFilter = {
+  MY_POSTS: 'my_posts',
+  MY_DISCUSSIONS: 'my_discussions',
+  MY_QUESTIONS: 'my_questions',
+};
+
+export const AllPostsFilter = {
+  ALL_POSTS: 'all_posts',
+  ALL_DISCUSSIONS: 'all_discussions',
+  ALL_QUESTIONS: 'all_questions',
+};
+
+
+export const TopicsFilter = {
+  ALL: 'all_topics',
+  COURSE_SECTION: 'course_section_topics',
+  GENERAL: 'general_topics',
+};
+
+
+export const Routes = {
+  TOPICS: {
+    PATH: '/courses/:courseId/discussion/forum',
```

Comment:
Probably `courses` there should be in singular form, like `/course/:courseId/discussion/forum`? 

Otherwise, paths from this MFE can conflict with paths from platform when they will be under the same domain.  [Same in Learning MFE](https://github.com/edx/frontend-app-learning/blob/c0d089563054a3277a33c4ef918c650893ee5618/src/index.jsx#L37).

Reply:
This is on purpose. To keep existing discussions URLs intact.